common/src/main/java/com/wynntils/screens/chattabs/ChatTabEditingScreen.java

```diff
@@ -231,41 +254,50 @@ public void doRender(GuiGraphics guiGraphics, int mouseX, int mouseY, float part
 
         PoseStack poseStack = guiGraphics.pose();
 
+        dividedWidth = this.width / GRID_DIVISIONS;
+        dividedHeight = this.height / GRID_DIVISIONS;
```

Comment:
I don't think these are required, doing it in the doInit should be enough

Reply:
Yes, these never change render time, but only when re-inited.

Reply:
Is init called when the window is resized? I'm pretty sure all the other screens also recalculate this in render

Reply:
it is, since widgets also need to be readded at correct locations

Reply:
I didn't recalculate it in the poi manager screen and I'm pretty sure it works